core/drivers/crypto/caam/crypto.mk

```diff
@@ -23,13 +32,66 @@
 # DBG_ECC    BIT32(12) // ECC trace
 # DBG_DH     BIT32(13) // DH Trace
 # DBG_DSA    BIT32(14) // DSA trace
-CFG_DBG_CAAM_TRACE ?= 0x2
+
+CFG_DBG_CAAM_TRACE ?= BIT32(1)
 CFG_DBG_CAAM_DESC ?= 0x0
 CFG_DBG_CAAM_BUF ?= 0x0
 
 # Enable the BLOB module used for the hardware unique key
 CFG_NXP_CAAM_BLOB_DRV ?= y
 
+ifeq ($(CFG_LS),y)
```

Comment:
I see two `ifeq ($(CFG_LS),y)` in this file. Could we get rid of this one? Maybe use `PLATFORM_FLAVOR` for i.MX to align with Layerscape ?

Reply:
indeed, PLATFORM_FLAVOR is a sufficient discriminant.

Reply:
update done